Orm/Xtensive.Orm/Orm/Linq/Translator.Materialization.cs

```diff
@@ -60,7 +63,7 @@ public TranslatedQuery Translate()
         projection.TupleParameterBindings, tupleParameterBindings);
 
       // Providing the result to caching layer, if required
-      if (compiledQueryScope != null && translatedQuery.TupleParameters.Count == 0) {
+      if (compiledQueryScope != null && !translatedQuery.TupleParameters.Any()) {
```

Comment:
Because of this line:
https://github.com/DataObjects-NET/dataobjects-net/pull/181/files#diff-ebcd93a0a53ec6c6c99779f61a2db28e4b8ae60e519315618d27d9adb5a3ad25R186

`translatedQuery.TupleParameters` is not always `List<>`. It save some allocations

Reply:
Here is the discussion
https://github.com/servicetitan/dataobjects-net/pull/40#discussion_r726437537

Reply:
Ok, my bad, resolved